components/script/dom/webgl_validations/tex_image_3d.rs

```diff
@@ -60,11 +59,7 @@ impl WebGLValidator for CommonTexImage3DValidator<'_> {
             .active_texture_for_image_target(target);
         let limits = self.context.limits();
 
-        let max_size = if target.is_cubic() {
-            limits.max_cube_map_tex_size
-        } else {
-            limits.max_tex_size
-        };
+        let max_size = limits.max_cube_map_tex_size;
```

Comment:
This doesn't match `where max is the returned value of GL_MAX_3D_TEXTURE_SIZE.` and other uses of max_size. I think you need a new field like https://github.com/servo/servo/blob/main/components/webgl/webgl_limits.rs#L16